File: 9. Palindrome Number/9.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        pal = str(x)

        if len(pal) == 1:
            return True
        for i in range(len(pal)//2):
            if pal[i] != pal[-i-1]:
                return False
        return True

File: 9. Palindrome Number/9.cs
public class Solution {
    public bool IsPalindrome(int x) {
        string str = x.ToString();
        if (str.Length == 1)
            return true;
        char[] charArray = str.ToCharArray();
        for (int i = 0; i< charArray.Length / 2; i++) {
            if (charArray[i] != charArray[charArray.Length - 1 - i])
                return false;
        }
        return true;
    }
} // Not the most efficient solution but straightforward to understand